src/main/java/drug_request_use_case/DrugRequestDsGateway.java

```diff
@@ -0,0 +1,6 @@
+package drug_request_use_case;
+
+public interface DrugRequestDsGateway {
+    boolean drugNameExists(String identifier);
+    void saveDrugRequest(DrugRequestDsInvokeModel drugRequestDsInvokeModel);
```

Comment:
Are we saving it to a database of "receipts" or something else?

Reply:
I haven't implemented how we should go about saving it yet but I'm prety sure it will be a CSV databse file.